iree/hal/metal/metal_driver.mm

```diff
@@ -16,10 +16,14 @@
 
 #import <Metal/Metal.h>
 
+#include "absl/flags/flag.h"
 #include "iree/base/status.h"
 #include "iree/base/tracing.h"
+#include "iree/hal/metal/metal_capture_manager.h"
 #include "iree/hal/metal/metal_device.h"
 
+ABSL_FLAG(bool, metal_capture, false, "Enables capturing Metal commands.");
```

Comment:
we should be keeping flags out of these files and in the _driver_module.cc files instead (it was a mistake that the renderdoc flag ended up in vulkan_driver - that should be moved out of there and passed in via the VulkanDriver::Options struct instead).

this is important for size-sensitive builds as absl::Flags is like >100KB of binary size increase and we want to be able to compile it out.

Reply:
Good point. Makes sense. Moved.